css/index.css

```
@@ -101,17 +127,201 @@ p {
     justify-content: space-evenly;
 }
 
```

So as we discussed in our 1 on 1 it would have been better to place all the CSS properties that you would be re-using on these boxes in a single base class.
```suggestion
.middle-content .boxes .box {
    width: 12.5%;
    height: 100px;
    margin: 20px 2.5%;
    color: white;
    display: flex;
    align-items: center;
    justify-content: center;
    }
```
And then place only the property that is changing into a more specific secondary class:

```suggestion
.middle-content .boxes .box1 {
background: teal;
}
```

That way you're keeping your code much more DRY. You're re-using the same code over and over again the way you have it now.